config/config-hd.sh.sample

```diff
@@ -50,20 +50,4 @@ CONFIRM_REBOOT=1
 
 CMD_GRUB_INSTALL="/bin/bash /sbin/grub-install"
 
-######################################################################
-# Define some debug output variables
-
-# Debug Levels - fixed values
-DEBUG_SILENT=0
-DEBUG_CRIT=1
-DEBUG_WARN=2
-DEBUG_INFO=4
-DEBUG_VERBOSE=7
-
-# Set your default debug level
-: ${DEBUG_DEFAULT:=${DEBUG_INFO}}
-
-# Dynamic debug level
-DEBUG_LEVEL=${DEBUG_DEFAULT}
-
-: ${TRACE:=0}
+DEBUG_LEVEL=${DEBUG_VERBOSE}
```

Comment:
I'd just remove this as well.  Since just leaving that line, without the DEBUG* variables in the same file, makes someone think "what are the valid values".

Leave a commented out assignment:

# DEBUG_LEVEL=${DEBUG_VERBOSE}

With a comment above it, indicating where the debug level options can be found.